### PR TITLE
cmd/web3: wire in existing interrupt cancellation; fix ASCII art alignment

### PR DESCRIPTION
**Pull request by jmank88**
Before:
```sh
  ___  _____  ___  _   _    __    ____  _  _ 
/ __)(  _  )/ __)( )_( )  /__\  (_  _)( \( )
( (_-. )(_)(( (__  ) _ (  /(__)\  _)(_  )  ( 
\___/(_____)\___)(_) (_)(__)(__)(____)(_)\_)
```
After:
```sh
  ___  _____  ___  _   _    __    ____  _  _ 
 / __)(  _  )/ __)( )_( )  /__\  (_  _)( \( )
( (_-. )(_)(( (__  ) _ (  /(__)\  _)(_  )  ( 
 \___/(_____)\___)(_) (_)(__)(__)(____)(_)\_)
```
screenshot from 2019-02-14 16-39-00

### CODE REVIEW

File: cmd/web3/start.go [hunk 1]
```diff
@@ -87,33 +85,9 @@ func start(c *cli.Context) error {
...
-		log.Fatalln("Starting command failed:", err)
-	}
-
-	done := make(chan error, 1)
-	go func() {
-		done <- cmd.Wait()
-	}()
-	// catch ctrl-c and kill
-	sigC := make(chan os.Signal, 2)
-	signal.Notify(sigC, os.Interrupt, syscall.SIGTERM)
```
Comment (treeder) [inline cmd/web3/start.go]: Why remove all these? We want to catch the no?

File: cmd/web3/main.go [hunk 2]
```diff
@@ -245,7 +245,9 @@ func main() {
 					EnvVar: pkVarName,
 				},
 			},
-			Action: start,
+			Action: func(c *cli.Context) error {
+				return start(ctx, c)
```
Comment (treeder) [inline cmd/web3/main.go]: What's the benefit of this change?

File: cmd/web3/start.go (hunk 1 above)
Comment (jmank88) [reply cmd/web3/start.go]: It is already handled in main. This is duplicate code. https://github.com/gochain-io/web3/pull/55#discussion_r255975063

File: cmd/web3/main.go (hunk 2 above)
Comment (jmank88) [reply cmd/web3/main.go]: The ctx is already cancelled on signal interrupts.

Comment (treeder) [reply cmd/web3/main.go]: Really?  Since when?

File: cmd/web3/start.go (hunk 1 above)
Comment (treeder) [reply cmd/web3/start.go]: This is required to pass the signal on to the sub process.

File: cmd/web3/main.go (hunk 2 above)
Comment (jmank88) [reply cmd/web3/main.go]: Not by default. It is already set up in main: https://github.com/gochain-io/web3/blob/master/cmd/web3/main.go#L43-L52

File: cmd/web3/start.go (hunk 1 above)
Comment (jmank88) [reply cmd/web3/start.go]: From the [godoc](https://golang.org/pkg/os/exec/#CommandContext) for `exec.CommandContext`:

> The provided context is used to kill the process (by calling os.Process.Kill) if the context becomes done before the command completes on its own.

File: cmd/web3/main.go (hunk 2 above)
Comment (treeder) [reply cmd/web3/main.go]: Ahh, I see, didn't catch that.

**Pull request merged**